src/rtl_airband.h

```diff
@@ -156,6 +152,16 @@ struct file_data {
 	enum output_type type;
 };
 
+struct udp_stream_data {
+	bool continuous;
+	const char *dest_ip;
+	int dest_port;
+
+	int send_socket;
+	struct sockaddr_in dest_addr;
+	int dest_addr_len;
```

Comment:
This variable is passed to `memset` and `sendto` which expect a `size_t`.

Reply:
looks like `sendto()` actually wants this as a `socklen_t`

Reply:
Indeed. I looked up the type of `len` rather than `addrlen`.